netdata.spec.in

```diff
@@ -42,9 +54,13 @@ Requires(pre): /usr/sbin/groupadd
 Requires(pre): /usr/sbin/useradd
 
 %if %{with systemd}
+%if 0%{?suse_version}
+%{?systemd_requires}
+%else
```

Comment:
No, I've had a look through `macros.perl` but I'm still unsure how to apply the conditional.